signs-work-order-map/app/page.tsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+import { useCallback, useState } from "react";
+import styles from "./page.module.css";
+import MapGL, { Marker, ViewStateChangeEvent } from "react-map-gl/mapbox";
+
+import GeocoderControl from "@/components/MapGeocoderControl";
+
+import {
+  DEFAULT_MAP_PARAMS,
+  DEFAULT_MAP_PAN_ZOOM,
+  MAP_COORDINATE_PRECISION,
+} from "@/config/map";
+
+interface LatLon {
+  latitude: number;
+  longitude: number;
+}
+
+export default function Home() {
```

Comment:
assuming this page is going to grow, might be good to move the map into it's own component now?